website/server/models/user/schema.js

```diff
@@ -137,6 +137,7 @@ let schema = new Schema({
       gemsBought: {type: Number, default: 0},
       mysteryItems: {type: Array, default: () => []},
       lastBillingDate: Date, // Used only for Amazon Payments to keep track of billing date
+      subscriptionLengthMonths: Number, // Length of Subscription
```

Comment:
Is this even really necessary? Don't we get the number of months from the subscription block, which we can find from the plan id?